language/move-model/src/building/exp_rewriter.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::{BTreeMap, BTreeSet, VecDeque};
+
+use crate::{
+    ast::Exp,
+    building::module_builder::ModuleBuilder,
```

Comment:
I wonder if "builder" is a better name than "building".  I don't have any deep reason for it. "building" just seemed odd to me.

Reply:
It does not work because there is already a builder.rs. I next tried `build` but this also does not work because it is in .gitignore ;-)

Reply:
> It does not work because there is already a builder.rs. I next tried `build` but this also does not work because it is in .gitignore ;-)

Found some renaming schema so its now `builder` as it supposed to be.